### PR TITLE
December board meeting terse notes

### PR DESCRIPTION
**Pull request by rrrutledge**

### CODE REVIEW

**Review by spier — APPROVED** [review]

File: content/en/about/board/2022-12-22.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Johannes Tigges
+Sebastian Spier
+Klaas-Jan Stol
+Georg Grütter
+Jacob Green
+Silona Bonewald (Treasurer)
+
+#### Directors and Officers Absent
+
+Thomas Sadler (Secretary)
```
Comment (lenucksi) [inline content/en/about/board/2022-12-22.md]: Wasn't @tsadler1988 present?

**Review by lenucksi — COMMENTED** [review]
Looks good, small question.

File: content/en/about/board/2022-12-22.md (hunk 1 above)
Comment (rrrutledge) [reply content/en/about/board/2022-12-22.md]: Oh that’s right - he came in after it had started.

Comment (lenucksi) [reply content/en/about/board/2022-12-22.md]:
```suggestion
```

File: content/en/about/board/2022-12-22.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Danese Cooper (Chair)
+Isabel Drost-Fromm (President)
+Daniel Izquierdo Cortazar (Vice President)
+Russell Rutledge (Assistant Secretary)
+Johannes Tigges
+Sebastian Spier
+Klaas-Jan Stol
+Georg Grütter
+Jacob Green
+Silona Bonewald (Treasurer)
```
Comment (lenucksi) [inline content/en/about/board/2022-12-22.md:28]: Added the change as per https://github.com/InnerSourceCommons/innersourcecommons.org/pull/439/files#r1061419461

```suggestion
Silona Bonewald (Treasurer)
Thomas Sadler (Secretary) (joined after meeting was called to order)
```

**Pull request merged**